### PR TITLE
Add support for -q argument to Manubulon snmp-storage commmand

### PR DESCRIPTION
**Pull request by markleary**
The -q argument specifies the storage type. Setting this to parameter to FixedDisk makes it easy to create a single service which monitors all fixed disks, ignoring NFS mounts for example.

### CODE REVIEW

File: itl/command-plugins-manubulon.conf [hunk 1]
```diff
@@ -215,6 +215,10 @@ object CheckCommand "snmp-storage" {
 			value = "$snmp_storage_olength$"
 			description = "Max-size of the SNMP message, usefull in case of Too Long responses."
 		}
+    "-q" = {
```
Comment (Crunsher) [inline itl/command-plugins-manubulon.conf]: Please indent following the rest of the file (tabs)

Comment (markleary) [reply itl/command-plugins-manubulon.conf]: Fixed indents and updated docs.

**Comment by Crunsher** [comment]
A'ight, looks good

**Comment by dnsmichi** [comment]
Thanks 👍

**Pull request merged**